recipes/braggedgemodeling/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "braggedgemodeling" %}
+{% set version = "0.1.1a2" %}
+{% set git_rev = "v0.1.1a2" %}
```

Comment:
Our bot won't be able to update your package this way. You can set `version` and then just leave the `v` in the URL below removing he need for this extra variable.